movie.sh

```diff
@@ -1,6 +1,60 @@
 #!/bin/bash
-echo 'please input the movie name'
-read name
-echo $name
-var=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$name | jq -r '.imdbRating')
-echo $var
+#Scan input directory from terminal
+
+
+if [ -z $1 ];
+then 
+    echo 'Directory not provided. Searching in current directory'
+    dir='.'
+else 
+    echo 'Directory is' $1
+    dir=$1
+fi
+
+#Get absolute path from the input directory
+path=$(find ~/ -name $dir)
+
+#Make a new file in tmp folder
+touch /tmp/sorted.txt || { echo "Failed to create temp file"; exit 1; }
+
+#Empty the sorted folder
+> /tmp/sorted.txt
+
+#Declaration of variables
+OUT="/tmp/sorted.txt"
+SPACE="   ";
+NA="NA";
+NULL="null";
+EMPTY="";
+ZERO="0"; 
+
+#Print the heading first
+printf "%-10s  %-20s  %-20s\n" "RATING" "MOVIE" "TITLE"
+
+for f in $path/*
+  do
+     #strip initial path value to get the movie name
```

Comment:
The number of spaces here is 5. Be consistent throughout the project. 2 spaces are preferred.